baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -161,17 +287,18 @@ def handle(self, message: kombu.Message) -> None:
                 self.handler_fn(context, message_body, message)
         except Exception as exc:
             prometheus_success = "false"
-            logger.exception(
-                "Unhandled error while trying to process a message.  The message "
-                "has been returned to the queue broker."
-            )
+
+            # Custom error_handler_fn has priority over standard handler.
             if self.error_handler_fn:
+                logger.debug(
+                    "Unhandled error while trying to process a message. Custom handler invoked."
+                )
                 self.error_handler_fn(context, message_body, message, exc)
             else:
-                message.requeue()
+                self._handle_error(message, prometheus_labels, exc)
 
             if isinstance(exc, FatalMessageHandlerError):
-                logger.info("Recieved a fatal error, terminating the server.")
+                logger.info("Received a fatal error, terminating the server.")
```

Comment:
What sort of error will be considered fatal?

Reply:
It's a pre-existing behavior; I have no idea which error any user of this framework can consider fatal.